hw_30_plan/js/script.js

```diff
@@ -0,0 +1,132 @@
+const listItems = [
+    'Work out',
+    'Get some sleep',
+    'Call the boyfriend',
+    'Cook the dinner',
+    'Watch Netflix',
+    'Take a shower',
+    'Walk with the dog',
+]
+
+let ul = document.querySelector('#ul');
+
+let menu = document.querySelector('#menu');
+
+class Menu {
+    constructor(elem) {
+        this._elem = elem;
+        elem.onclick = this.onClick.bind(this);
+    }
+
+    addStart() {
+        let newItem = document.createElement('li');
+        newItem.textContent = listItems[Math.floor(Math.random() * listItems.length)];
+        ul.prepend(newItem);
+    }
+
+    addEnd() {
+        let newItem = document.createElement('li');
+        newItem.textContent = listItems[Math.floor(Math.random() * listItems.length)];
+        ul.append(newItem);
+    }
+
+    delete() {
+        let selectedItems = Array.from(ul.children).filter(item => item.classList.contains('selected'));
+        selectedItems.forEach(item => item.remove());
+    }
+
+    sort() {
+        let selectedItems = Array.from(ul.children).filter(item => item.classList.contains('selected'));
+        selectedItems.reverse().forEach(item => ul.prepend(item));
+    }
```

Comment:
(Don't Repeat Yourself). You have 2 identical values `selectedItems` in methods delete() and sort(). It's better to put them in a separate method. Also `Array.from(ul.children).filter(item => item.classList.contains('selected'));` can be shortened as `Array.from(ul.querySelectorAll(".selected"));`